lib/ReactTags.js

```diff
@@ -107,15 +115,19 @@ class ReactTags extends Component {
     }
   }
 
-  filteredSuggestions(query, suggestions) {
+  filteredSuggestions = (query, suggestions) => {
```

Comment:
We don't need arrow function here. Can this be removed ?

Reply:
Probably. The only reason I did it like this is because you said you didn't want to bind it and to use arrow functions instead.

Reply:
yeah but binding was also not needed here thats why adding arrow function was not helping. It was needed here https://github.com/josh-degraw/react-tags/blob/1a9ad5470ceb98a0584fb5f80964146b0b4cb350/lib/ReactTags.js#L123 which is fine.